src/_utils/__tests__/ArrayUtils-test.js

```diff
@@ -1,7 +1,7 @@
 import expect from 'expect';
-import { groupByKey } from '../ArrayUtils';
+import { groupByKey,objectToArray } from '../ArrayUtils';
 
-describe('groupByKey', () => {
+describe('ArrayUtils', () => {
```

Comment:
Make it consistent, if the test is only about groupByKey, then `describe('groupByKey')`